app/views/pages/index.html.erb

```diff
@@ -9,5 +9,5 @@
 </h3>
 <%= render "header" %>
 
-<%= react_component('RouterApp', props: render(template: "/comments/index.json.jbuilder"),
```

Comment:
Note how we removed the component props initialization here since we are using shared redux stores.